common/src/main/java/com/microsoft/identity/common/internal/net/cache/HttpCache.java

```diff
@@ -63,7 +63,7 @@
     public static synchronized boolean initialize(@NonNull final File cacheDirectory,
                                                   @NonNull final String cacheFileName,
                                                   final long maxSizeBytes) {
-        final String methodName = ":initialize (File, Filename, Capacity)";
+        final String methodName = ":initialize(File, Filename, Capacity)";
```

Comment:
we can remove this

Reply:
Here: da1e62d1ddacd758eb5d0a6cedd9589521475ab8